.history/popup_20230518181610.js

```diff
@@ -0,0 +1,199 @@
+const body = document.querySelector('body');
+// const overlay = document.querySelector('.overlay');
```

Comment:
- [ ] [OPTIONAL] As I mentioned earlier, I think removing this will make your work look like the professional that you are. Consider removing it, please :pray: